packages/contentstack-bootstrap/package.json

```diff
@@ -17,7 +17,7 @@
     "test:report": "nyc --reporter=lcov mocha \"test/**/*.test.js\""
   },
   "dependencies": {
-    "@contentstack/cli-cm-seed": "~1.6.1",
+    "@contentstack/cli-cm-seed": "~1.6.2",
```

Comment:
Why did we bump the version is seed ? why didnt we bump it in export-to-csv, Pls update the contentstack package json as well with the version bump

Reply:
seed was using utilities that's why bumped it's version also. Contentstacks version was also bumped I updated the branch with development changes that's why its not showing now

Reply:
what about "export-to-csv", is it bumped already